src/app/hearing/components/view-hearing/view-hearing.component.html

```diff
@@ -9,7 +9,8 @@
     </div>
       <mat-select #actionSelect id="action" value="{{hearingActions.Actions}}" fxFlex="10" (selectionChange)="onActionChanged($event)">
         <mat-option [value]="hearingActions.Actions">{{hearingActions.Actions}}</mat-option>
-        <mat-option [value]="hearingActions.Unlist" [disabled]="!isListed()">{{hearingActions.Unlist}}</mat-option>
+        <mat-option [value]="hearingActions.Unlist" [disabled]="!hearing.possibleActions.Unlist">{{hearingActions.Unlist}}</mat-option>
```

Comment:
I feel a conflict in the air 🎵

Reply:
I agree, it is a bit confusing at first glance - although there is another approach taken in SL-1900 which will be used over this (i.e. its going to be replaced very shortly)